src/frontend/efiling-frontend/nginx.conf

```diff
@@ -10,21 +10,6 @@ server {
   location ~* /efilinghub/\.env-config\.js$ {
     expires -1;
     add_header 'Cache-Control' 'no-store, no-cache, must-revalidate, proxy-revalidate, max-age=0';
-    add_header X-Content-Type-Options "nosniff" always;
-    add_header X-XSS-Protection "1; mode=block";
-    add_header X-Content-Type-Options "nosniff" always;
-    add_header Cache-Control "no-store";
-    add_header "Pragma" "no-cache";
-    add_header Content-Security-Policy "default-src 'self';";
-    add_header Strict-Transport-Security "max-age=31536000; includeSubDomains" always;
-  }
-  
-  location /images {
-    deny all;
-  }
-  
-  location /static {
-    deny all;
```

Comment:
Same. remove nginx changes from this PR to keep UI and DevOps changes separate - easier to track.